java/src/jmri/jmrix/can/cbus/node/CbusNodeStats.java

```diff
@@ -113,9 +113,9 @@ public int totalRemainingNodeBytes(){
      * @return float min 0 max 1
      */
     public float floatPercentageRemaining(){
-        float remain = ( 1.0f * ( totalNodeBytes() - totalRemainingNodeBytes() ) ) / ( totalNodeBytes() );
-        if ( remain > 0 && remain < 1.000001 ) {
-            return remain;
+        float soFar = ( 1.0f * ( totalNodeBytes() - totalRemainingNodeBytes() ) ) / ( totalNodeBytes() );
+        if ( soFar > 0 && soFar < 1.000001 ) {
```

Comment:
Would a return value of 1.0000001 mess up comparisons in other places?  If so, might be good to return `Math.min(soFar, 1.0)`. No problem if not.